projects/flow.groups-2D-random-walk/README.md

```diff
@@ -0,0 +1,68 @@
+A study on 2D Gaussian random walks using **groups** and **aggregates**
+
+Here we use simulations of a 2D random walk to demonstrate the **groups** and **aggregation** features of **signac-flow**.
+In addition, this example tries to showcase a robust workflow similar to those of real research projects while keeping the actual computation at a low level.
+
+# Usage
+
+1. Initialize the project with
+
+    ```
+    python3 src/init.py
+    ```
+
+2. Now let's simulate the random walks:
+
+    ```
+    python3 src/project.py run -o simulate
+    ```
+
+    Alternatively, we can use the base group to do some analysis as well with
+
+    ```
+    python3 src/project.py run -o base
+    ```
+
+    To see all the operations this previous command runs look at the `project.py` file.
+
+
+3. From here we can use the `post-processing` group to run all the aggregate operations (besides one) computing the mean squared displacement, and creating multiple plots.
+
+    ```
+    python3 src/project.py run -o post-processing
+    ```
+
+4. Now we can run the final two operations, `plot_mean_squared_distance` and `plot_walks` with
+    ```
+    python3 src/project.py run -o plot_mean_squared_distance plot_walks
+    ```
+
+The current status of the project can be viewed using
+
+```
+python3 src/project.py status -d
+```
+
+If `signac_dashboard` is installed, a dashboard can be viewed with the following command,
```

Comment:
I installed signac dashboard with `python3 -m pip install signac_dashboard`.

Also, probably, although this will fail first since it asks for `signac-flow=v.0.15` since aggregation is not released yet.